src/app/person-search/components/person-search-header/person-search-header.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+import { StudentsRestService } from '../../../shared/services/students-rest.service';
+import { Router } from '@angular/router';
+import { DropDownItem } from '../../../shared/models/drop-down-item.model';
+
+export interface StudentFilter {
+  student: Option<number>;
+}
+
+@Component({
+  selector: 'erz-person-search-header',
+  templateUrl: './person-search-header.component.html',
+  styleUrls: ['./person-search-header.component.scss'],
+})
+export class PersonSearchHeaderComponent {
+  constructor(
+    public studentsRestService: StudentsRestService,
+    private router: Router
+  ) {}
+
+  filter: StudentFilter = {
+    student: null,
+  };
+
+  navigateToDossier(key: DropDownItem['Key']) {
+    const id = Number(key);
+    this.router.navigate(['person-search', 'student', id, 'addresses']);
```

Comment:
Musst du hier auf die `addresses` springen? Solltest du nicht einfach auf die Dossier Default Route gehen? `this.router.navigate(['person-search', 'student', id);`?

Reply:
Wird gemacht, damit einer der drei Tabs auf dem Dossier bereits selektiert ist, wenn der Benutzer darauf navigiert.